packages/api-client/src/components/ImportCollection/utils/isUrl.ts

```diff
@@ -0,0 +1,4 @@
+/** Checks whether the given string is an URL */
+export function isUrl(input: string | null) {
+  return input && (input.startsWith('http://') || input.startsWith('https://'))
+}
```

Comment:
would it make send to use a url regex as a checker in the regexHelpers?

Reply:
Hm … I think it’s fine. Way easier to read than any regex and not performance relevent.